src/osmLayer.js

```diff
@@ -139,6 +142,18 @@ let StamenAttribution = 'Map tiles by <a href="http://stamen.com">Stamen ' +
  * @type {object}
  */
 osmLayer.tileSources = {
+  'darkMatter-with-labels': {
```

Comment:
For consistency, should this be `dark-matter-with-labels` rather than camel case?

Reply:
Yes, yes it should.